docs/topics/crud-mission-deploy-booster.adoc

```diff
@@ -44,14 +44,27 @@ my-database-1-deploy   0/1       Completed   0         53s
 +
 Your `my-database` pod should have a status of `Running` and should be indicated as ready once it is fully deployed and started.
 
-. Use maven to start the deployment to your {OpenShiftLocal}.
+ifndef::http-api-nodejs[]
+. Use maven to start the deployment to {OpenShiftLocal}.
 +
 [source,bash,options="nowrap",subs="attributes+"]
 ----
 $ mvn clean fabric8:deploy -Popenshift
 ----
 +
 This command uses the Fabric8 Maven Plugin to launch the S2I process on your {OpenShiftLocal} and to start the pod.
+endif::http-api-nodejs[]
+
+ifdef::http-api-nodejs[]
+. Use `npm` to start the deployment to {OpenShiftLocal}.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm install && npm run openshift
+----
++
+These commands use the Nodeshift `npm` module to install your dependencies and launch the S2I process on your {OpenShiftLocal} and start the pod.
```

Comment:
I think we changed this formulation recently (?). This sounds like Nodeshift is used to install dependencies, which isn't exactly correct.

Reply:
@Ladicek tracked in #557

Reply:
Ah, thanks!